packages/tools/src/tools/CrosshairsTool.ts

```diff
@@ -181,6 +181,28 @@ class CrosshairsTool extends AnnotationTool {
       defaultReferenceLineSlabThicknessControlsOn;
   }
 
+  resetAnnotations = () => {
```

Comment:
can't you disable and enable again the tool?

Reply:
At least add an example with resetting in the tool samples, but also try enable/disable to see what happens.